### PR TITLE
Add progress bar

### PR DESCRIPTION
**Pull request by aleitner**

### CODE REVIEW

**Comment by mniewrzal** [comment]
I run cp with progress and my output is:
`
305.86 MiB / 316.31 MiB [=================================================================================================================================>----]  96.69%
Downloaded s3://abcd/case-3.mp4 to /home/wywrzal/test/copies/a.mp4
`

File: go.mod [hunk 1]
```diff
@@ -163,6 +164,7 @@ require (
 	gopkg.in/Shopify/sarama.v1 v1.17.0 // indirect
 	gopkg.in/VividCortex/ewma.v1 v1.1.1 // indirect
 	gopkg.in/alecthomas/kingpin.v3-unstable v3.0.0-20180810215634-df19058c872c // indirect
+	gopkg.in/cheggaaa/pb.v1 v1.0.25
 	gopkg.in/cheggaaa/pb.v2 v2.0.6 // indirect
```
Comment (egonelbre) [inline go.mod]: Why are both v1 and v2 being imported?

Comment (aleitner) [reply go.mod]: ¯\_(ツ)_/¯ go did that not me

Comment (egonelbre) [reply go.mod]: When you use `pb.v2` then it would use the same dependency, hence not import v1. I think v2 might be coming from viper/cobra.

**Comment by brimstone** [comment]
How's this look now @egonelbre ?

**Pull request merged**